components/scream/src/control/tests/sc_tests.cpp

```diff
@@ -253,6 +253,7 @@ TEST_CASE ("recreate_mct_coupling")
   FL scalar2d_layout{ {COL          }, {ncols          } };
   FL vector2d_layout{ {COL, CMP     }, {ncols, 2       } };
   FL scalar3d_layout{ {COL, LEV     }, {ncols,    nlevs} };
+  FL scalar3d_int_layout{ {COL, LEV     }, {ncols,    nlevs+1} };
```

Comment:
I think we have a tag `ILEV` for when layouts are defined at the interface instead of midpoint.  So this should be `{COL, ILEV}`

Reply:
That's my bad. I'll fix it.